src/modules/shareddrives/hooks/useSharedDriveFolder.tsx

```
@@ -1,50 +1,48 @@
-  // return {
-  //   sharedDriveQuery,
-  //   sharedDriveResult
-  // }
-
   const client = useClient()
   const [sharedDriveResult, setSharedDriveResult] = useState<
     SharedDriveFolderReturn['sharedDriveResult']
   >({ data: undefined })
+  const [fetchStatus, setFetchStatus] = useState<
+    SharedDriveFolderReturn['fetchStatus']
+  >('loading')
```

ESLint

[error] Replace `·useState<⏎····SharedDriveFolderReturn['fetchStatus']⏎··` with `⏎····useState<SharedDriveFolderReturn['fetchStatus']`

(prettier/prettier)